tutorials/notebooks/imx500_notebooks/pytorch/pytorch_efficient_anomaly_detection.ipynb

```diff
@@ -0,0 +1,824 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xGqjJDA1AaJo"
+      },
+      "source": [
+        "# Anomaly Detection Training Benchmark and Quantization for IMX500"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jHiBhX_DkkGH"
+      },
+      "source": [
+        "[Run this tutorial in Google Colab](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/imx500_notebooks/pytorch/pytorch_efficient_anomaly_detection.ipynb)\n",
+        "\n",
+        "### Overview\n",
+        "\n",
+        "In this tutorial we demonstrate training, quantization and benchmarking of an anomaly detection model. The resulting model will be imx500 compatible"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LF92AFYDJGd0"
+      },
+      "source": [
+        "Classification models are powerful and reliable, but what if you have little or no examples of one of your classes, what if one of your classes contains too much unpredictable variation?\n",
+        "\n",
+        "Here we go through the process of building, training and quantising an anomaly detection model designed to solve exactly these problems.\n",
+        "\n",
+        "Anomaly detection models are useful as they only require your typical images to train and can in theory determin anything that is not typical.\n",
+        "\n",
+        "We use Efficient ad, one of the top performing anomaly detection models on the mvtec benchmark. benchmark leader board can be found [here](https://paperswithcode.com/sota/anomaly-detection-on-mvtec-ad)\n",
```

Comment:
We use *Efficient ad* --> add link
on the *mvtec benchmark* --> add link (if available)